extensions/toolbox/src/script/date/utc.ts

```diff
@@ -0,0 +1,26 @@
+import { Script } from "../type";
+
+export const toUtc: Script = {
+  info: {
+    title: "Date to UTC",
+    desc: "Converts dates and timestamps to UTC dates",
```

Comment:
`Convert dates and timestamps to UTC dates` would be better here.

Reply:
Thank you. I fixed it.